contracts/deploy/mainnet/122_simple_harvester_v2.js

```diff
@@ -0,0 +1,177 @@
+const addresses = require("../../utils/addresses");
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "122_simple_harvester_v2",
+    forceDeploy: false,
+    //forceSkip: true,
+    reduceQueueTime: true,
+    deployerIsProposer: false,
+    proposalId: "",
+  },
+  async ({ deployWithConfirmation, withConfirmation }) => {
+    const { multichainStrategistAddr, deployerAddr } = await getNamedAccounts();
+    const sDeployer = await ethers.provider.getSigner(deployerAddr);
+
+    // ---- Menu ---
+    // 1. Deploy new simple Harvester
+    // 2. Change harvester on all SSV strategies
+    // 3. Support strategies on the harvester
+    // 4. Sending all WETH from the OldDripper to FixedRateDripper
+    // 5. Update harvester on OETH AMO
+    // 6. Support AMO strategy on new harvester
+    // --------------
+
+    // 1. Deploy new simple Harvester
+    const cOETHFixedRateDripperProxy = await ethers.getContract(
+      "OETHFixedRateDripperProxy"
+    );
+
+    // 1.a Deploy proxy
+    const dOETHSimpleHarvesterProxy = await deployWithConfirmation(
+      "OETHSimpleHarvesterProxy"
+    );
+
+    const cOETHSimpleHarvesterProxy = await ethers.getContractAt(
+      "OETHSimpleHarvesterProxy",
+      dOETHSimpleHarvesterProxy.address
+    );
+
+    // 1.b Deploy implementation
+    const dOETHHarvesterSimpleImpl = await deployWithConfirmation(
+      "OETHHarvesterSimple",
+      [addresses.mainnet.WETH],
+      undefined,
+      true
+    );
+
+    const cOETHHarvesterSimpleImpl = await ethers.getContractAt(
+      "OETHHarvesterSimple",
+      dOETHHarvesterSimpleImpl.address
```

Comment:
nit: same as the previous comment, `getContract` can be used here

Reply:
Fixed: https://github.com/OriginProtocol/origin-dollar/pull/2363/commits/bc98ec8bb0f057a708c03ac195c0865b68a5c0ec.